### ISSUE TITLE
Bill types is not enabled in cashcode bill validator

### ISSUE THREAD

**Issue by sarathjasrin** [Desionlab/node-cashcode-bv#6]
I tried to connect cashcode bill validator using ccnet protocol implemented in node js. I tried to enable `billTypes` it's showing any error.  When I tried to print the `BillTypes` it's showing `enabled` as `false`.

        [ { amount: 1, code: 'QAT', enabled: false, security: false },
      { amount: 5, code: 'QAT', enabled: false, security: false },
      { amount: 10, code: 'QAT', enabled: false, security: false },
      { amount: 50, code: 'QAT', enabled: false, security: false },
      { amount: 100, code: 'QAT', enabled: false, security: false },
      { amount: 500, code: 'QAT', enabled: false, security: false }]

My bill validator model: MSM-QA1112

**Comment by fenixphp** [comment]
Sample code you used?

**Comment by sarathjasrin** [comment]
I used the example code from the repo.
```
const { CCNet } = require('cashcode-bv');
let device = new CCNet.BillValidator('COM3');
async function init (){
   await device.connect();
  console.log(device.info);
  console.log(device.billTable);
}
init()
.then(function () {
  console.log('Ok.');
})
.catch(function (error) {
  console.error(error);
});
```

**Comment by fenixphp** [comment]
Yes, there were problems with this; at the time of its creation the library did not have to correct this. In addition, the behavior of the device does not correspond to the documentation. In the second version it should be improved, but when I finish, I don’t know. There is no time allocated for this project. The new version is in the "refactoring" branch, but it is not ready.

**Comment by sarathjasrin** [comment]
First of all thanks for the project. You already put good amount of time creating this project hopefully you will finish soon. Till I have to wait and I have no choice.  God bless you.
